src/components/misc/MatomoSupport.ts

```diff
@@ -32,8 +32,13 @@ export const MatomoScript = () => {
 };
 
 export const MatomoTrackRegistration = () => {
-  window._paq = window._paq ?? [];
```

Comment:
I think this is still needed, as the Matomo Script is loaded asynchronously and the `window._paq` property might not be initialized yet

Reply:
The script with all the matomo client logic that we load from the cloud is run asynchronously but not the MatomoScript() function which we run in index.tsx. so theoretically it shouldn't be a problem or am I still mistaken?

Reply:
Ah true :)